Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -1357,7 +1443,7 @@ final class PledgeViewModelTests: TestCase {
     }
   }
 
-  func testOutputGoToRiskMessagingModal_OptimizelyClientVariant1() {
+  func testOutputGoToRiskMessagingModal_OptimizelyClientVariant1_StandardPledge() {
```

Comment:
Shouldn't this be `Control` because variant one means risk messaging would show.

Reply:
The variant is set here. I think the assertion at the bottom may have confused you because its checking for `false` here:
`self.goToRiskMessagingModal.assertValue(false)`


I'll add some documentation to the code but here the `false` implies it is a standard pledge and `true` would mean it is Pay With Apple.